src/modal.js

```diff
@@ -57,23 +57,27 @@ function getRootElement() {
 
 // A wrapper for react-modal that adds some styles and a close button.
 // See https://github.com/reactjs/react-modal for usage.
-function Modal({ onClose, hideCloseButton, children, ...rest }) {
+function Modal({ onClose, preventClose, children, ...rest }) {
+  const canClose = !preventClose
   return (
     <ReactModal
-      isOpen
+      isOpen={true}
```

Comment:
I follow the updated `preventClose` prop but not sure I follow why `isOpen` is always `true`? Seems like it can be overridden by the `{...rest}`.

Reply:
Would it be more clear to set `defaultProps` value for `isOpen` and then this line would be `isOpen={isOpen}`?

Reply:
@bhennes2 I'm not actually changing any functionality here. `isOpen` is syntactic sugar for `isOpen={true}`, since props default to true. However, the React documents actually recommend _against_ this (hence the change).

https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

Reply:
👍 👍

Reply:
I like your suggestion! I'll make that change in addition to the changes made for `preventClose` in this PR